docs/data.mdx

```diff
@@ -49,7 +50,7 @@ Any small enough tabular data files, such as tsv, csv, BED, BEDPE, and GFF, can
 
 ```
 
-<TableWrapper objName='CSVData' GoslingSchema={GoslingSchema} />
```

Comment:
Not sure why these all got renamed but I found that most of the object names had changed

Reply:
Yes! We, at some point, changed the letter cases of Data objects for consistency with other names.